.github/workflows/deploy-integration-tests.yaml

```diff
@@ -36,6 +36,35 @@ jobs:
       astronomer_key_secret: ${{ secrets.PROVIDER_INTEGRATION_TESTS_ASTRONOMER_KEY_SECRET }}
       SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
 
+  wait-for-providers-integration-tests-to-be-ready:
+    needs: deploy-to-providers-integration-tests
+    if: github.event_name == 'schedule'
```

Comment:
Can we reverse these conditions so that it becomes more readable that way? I mean run the GitHub action on schedule and then it would need `deploy-to-providers-integration-tests` before starting.

Reply:
Sure!